### ISSUE TITLE
Debian Bookworm Package?

### ISSUE THREAD

**Issue by glennbrown** [trapexit/mergerfs#1205]
Do you plan to release a Debian Bookworm package for the current release or will that come with the next release?

**Comment by trapexit** [comment]
Yes, at least with the next release. I don't generally watch distros and update older releases unless requested.

**Comment by trapexit** [comment]
https://github.com/trapexit/mergerfs/releases/download/2.35.1/mergerfs_2.35.1.debian-bookworm_amd64.deb
